sdk/core/core/src/az_http_request.c

```diff
@@ -20,22 +20,22 @@ AZ_NODISCARD az_result _az_is_question_mark(az_span slice)
 }
 
 AZ_NODISCARD az_result az_http_request_init(
-    _az_http_request* p_hrb,
```

Comment:
It is good we no longer use `hrb` naming. At a time, the struct was named `http_request_builder` so this is where `hrb` comes from; but now it is just `http_request`, so the correct name would be `request`, and in this specific function we should name it as `out_request` because we don't read it, but fill it in as an output parameter.

Reply:
Yes, I agree